gpytorch/kernels/grid_kernel.py

```diff
@@ -1,13 +1,14 @@
 #!/usr/bin/env python3
 
+from typing import List
+
 import torch
 from torch import Tensor
+
 from .kernel import Kernel
-from ..lazy import delazify, ToeplitzLazyTensor, KroneckerProductLazyTensor
 from .. import settings
-import gpytorch
-from gpytorch.utils.grid import create_data_from_grid, convert_legacy_grid
-from typing import List
+from ..utils.grid import create_data_from_grid, convert_legacy_grid
+from ..lazy import delazify, ToeplitzLazyTensor, KroneckerProductLazyTensor, cat as gpytorch_cat
 
 
 class GridKernel(Kernel):
```

Comment:
Now that Kernels define whether they are stationary, we can raise a RuntimeError in the construct of GridKernel if `base_kernel` is not stationary (it is required to be).

Reply:
Happy to add this this check in.